### PR TITLE
Add test node package

### PR DESCRIPTION
**Pull request by destogl**
After #71 is merged, this needs a rebase.

### CODE REVIEW

**Comment by bmagyar** [comment]
@destogl ready for rebase, once done, feel free to drop me a note to review this

**Comment by destogl** [comment]
@bmagyar Before merge please test it with the manual.

File: ros2_control_test_nodes/test/test_copyright.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
```
Comment (bmagyar) [inline ros2_control_test_nodes/test/test_copyright.py:1]: we have linters for this on the CI, let's use those please, I'd really prefer if we could get rid of junk like this

File: ros2_control_test_nodes/ros2_control_test_nodes/__init__.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Stogl Robotics Consulting UG (haftungsbeschränkt)
```
Comment (bmagyar) [inline ros2_control_test_nodes/ros2_control_test_nodes/__init__.py:1]: do we really need to litter in the init.py files too??

File: ros2_control_test_nodes/resource/ros2_control_test_nodes [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (bmagyar) [inline ros2_control_test_nodes/resource/ros2_control_test_nodes:1]: empty file, not intended?

**Review by bmagyar — CHANGES_REQUESTED** [review]
please don't forget to add the new package to the gh workflows

File: ros2_control_test_nodes/ros2_control_test_nodes/__init__.py (hunk 2 above)
Comment (destogl) [reply ros2_control_test_nodes/ros2_control_test_nodes/__init__.py:1]: Unfortunately, yes....

File: ros2_control_test_nodes/resource/ros2_control_test_nodes (hunk 3 above)
Comment (destogl) [reply ros2_control_test_nodes/resource/ros2_control_test_nodes:1]: It has to be there in ros2 python packages... No idea why. See, e.g., https://github.com/ros2/launch/blob/master/launch/resource/launch

**Comment by destogl** [comment]
@bmagyar just updated... Please one more check.

**Review by bmagyar — APPROVED** [review]

**Pull request merged**